x-pack/plugins/security_solution/public/landing_pages/pages/threat_hunting.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+import { SecurityPageName } from '../../app/types';
+import { HeaderPage } from '../../common/components/header_page';
+import { SecuritySolutionPageWrapper } from '../../common/components/page_wrapper';
+import { SpyRoute } from '../../common/utils/route/spy_routes';
+import { LandingLinksImages, NavItem } from '../components/landing_links_images';
+import * as i18n from './translations';
+import userPageImg from '../../common/images/users_page.png';
+import hostsPageImg from '../../common/images/hosts_page.png';
+import networkPageImg from '../../common/images/network_page.png';
+
+const items: NavItem[] = [
+  {
+    id: SecurityPageName.hosts,
+    label: i18n.HOSTS,
+    description: i18n.HOSTS_DESCRIPTION,
+    image: hostsPageImg,
+    imageAlt: i18n.HOSTS_IMAGE_ALT,
```

Comment:
We can remove the `imageAlt` property from the item config, it can be generated if we interpolate the label into it:
```
i18n.translate('x.x.x', {
    values: { pageName },
    defaultMessage:
      '{pageName} page screenshot',
  });
```